docs/GettingStarted.asciidoc

```diff
@@ -366,9 +366,8 @@ to play deeper, you can configure the whole openQA manually from scratch, but
 this document should help you to get started faster.
 
 === Getting tests
-Set `git_auto_clone = yes` in `openqa.ini`. Then you can point `CASEDIR` and
-`NEEDLES_DIR` to Git repositories. openQA will checkout those repositories
-automatically and no manual setup is needed.
+You can point `CASEDIR` and `NEEDLES_DIR` to Git repositories. openQA will
+checkout those repositories automatically and no manual setup is needed.
```

Comment:
How about mentioning how to disable it then?

Reply:
I decided against it because we shouldn't bother people with this in the "Getting started" section. Anybody who is interested in checking out what can be tweaked will find the setting with an explanation in the default config template.